modules/cardano-service.nix

```diff
@@ -174,14 +176,15 @@ in {
       # how many minutes between nodes restarting
       nodeMinute = mod (cfg.nodeIndex * 4) 60;
     in {
-      script = ''
-        /run/current-system/sw/bin/systemctl restart cardano-node
-      '';
+      script = (if (config.deployment.arguments.configurationKey == "bench")
+        then '' echo System not restarted because benchmark is running.''
+        else '' /run/current-system/sw/bin/systemctl restart cardano-node'');
       # Reboot cardano-node every 36h (except Mon->Tue gap which is 24h)
-      startAt = [
-        "Tue,Fri,Mon 13:${toString nodeMinute}"
-        "Thu,Sun     01:${toString nodeMinute}"
-      ];
+      #startAt = [
+      #  "Tue,Fri,Mon 13:${toString nodeMinute}"
+      #  "Thu,Sun     01:${toString nodeMinute}"
+      #];
+      startAt = lib.mkForce [];
```

Comment:
and revert the change to `startAt`, the `script` change can stay

Reply:
Done.